File: app/src/main/java/com/example/phonetennis2/MainActivity.java
package com.example.phonetennis2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button button1;
    private Button button2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button1 = (Button) findViewById(R.id.button1);
        button1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                openPlayer1();
            }
        });

        button2 = (Button) findViewById(R.id.button2);
        button2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                openPlayer2();
            }
        });
    }

    public void openPlayer1(){
        Intent intent1 = new Intent(this, Player1.class);
        startActivity(intent1);
    }

    public void openPlayer2(){
        Intent intent2 = new Intent(this, Player2.class);
        startActivity(intent2);
    }
}

File: app/src/main/java/com/example/tennis/MainActivity.kt
package com.example.tennis


import android.os.Bundle
import android.util.Log
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        Log.v(TAG, "testing...")
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val t: TextView = findViewById(R.id.words)
        t.text = "test"
    }

}